London | 25-SDC-Nov | Zohreh Kazemianpour | Sprint 1 | Individual -Shell -Tools

zohrehKazemianpour opened the pull request:
Self checklist

- [ x] I have titled my PR with Region | Cohort | FirstName LastName | Sprint | Assignment Title
- [x ] My changes meet the requirements of the task
- [x ] I have tested my changes
- [x ] My changes follow the [style guide](https://curriculum.codeyourfuture.io/guides/reviewing/style-guide/)

## Changelist

Completed all exercises for individual shell tools (ls, cat, wc, grep, sed, awk).

zohrehKazemianpour committed:
Complete ls exercises

zohrehKazemianpour committed:
Complete cat exercises

zohrehKazemianpour committed:
complete wc exercise

zohrehKazemianpour committed:
complete grep exercises

zohrehKazemianpour committed:
Complete sed exercises

zohrehKazemianpour committed:
Complete awk exercises

LonMcGregor reviewed (commented):
Good start on this sprint's tasks, I have spotted a few areas where you could improve code further

zohrehKazemianpour commented:
> Great, this PR looks complete now

Thanks for approving this PR! I just wanted to let you know that I found out the binary commit was accidentally in my main branch, which is why it appeared in all of my PRs. I've cleaned up main and rebased all the other open PRs to remove it. Then I noticed all of the commit times had changed to today's date.  I think that's because of the rebase rewriting the history.
